docs/portal/developer-portal/advanced/Enabling_K3s.md

```diff
@@ -208,7 +207,7 @@ uan_haproxy:
 ```
 This is an example that should be tailored to the desired configuration. See the [SSHD Configuration](#sshd-configuration) section to create new instances of SSHD to respond to HAProxy connections outside of the standard SSHD running on port 22.
 
-For more information HAProxy configurations, see [HAProxy Configuration](#https://docs.haproxy.org/2.7/configuration.html).
+For more information HAProxy configurations, see [HAProxy Configuration](<https://docs.haproxy.org/2.7/configuration.html>
```

Comment:
Looks like the closing ")" is missing from this link.

Reply:
fixed in second commit.